### ISSUE TITLE
Add documentation to repositories.

### ISSUE THREAD

**Issue by jamalv** [teamforus/foundation#4]
_initial draft_

**Problem statement:** 

More and more people are getting involved with Forus, sharing information is becoming more difficult to do just in conversations. This is starting to cause information discrepancies.

**Solution:**

Each repo should have a /docs folder with the information about the repository structured in such a way that any outsider can inform themselves. 

The text in the documentation can be quite high level / easy to understand, and link to different pages with a deeper explanation. I think it would be useful to write most of it ourselves, so that we can share our perspective. If readers find a subject interesting they can simply google it to learn more about it.

Below we can work on the outline; what goes where and what do we want to explain. Just add comments and I'll update the text below. When we have a good first draft, we will add it in a docs brach in the appropriate repository and continue work there.

# me

### what is this application?
* explanation about self sovereign identity in layman terms

### what technology is being used
* ethereum/ipfs etc. also explain centralised components

### standards
* not trying to reinvent the wheel, ERC20, ERC721, ERC725, ERC735, explain what they are in layman terms.

# forus.io

### what is this application?
* explain "platform for conditional exchange of value etc.

# research-and-development
* explain a lot of work is needed on learning how to decentralise. this is a way to invite anybody into this process and share the results.

# organisation
## about the organisation
* "Forus" comes from "for us" and is an open organisation, everybody is welcome to get involved and "join us".

## way of working
* we want to provide value right now, and also have a vision of "decentralising ourselves away" as much as possible.
* our vision about decentralisation, its current status and our roadmap, how we are planning to implement in Zuidhorn, etc.

## collaboration
* at some point we might do open zoom meetings etc, here we can have the schedules etc.

# foundation
* Foundation Forus is based in the Netherlands, mission "decentralised applications for the public domain" non-profit.
* Here we can share our statutes, yearly results, presentations and other media

# service
* We're working with partners to offer services that support decentralised applications, explain our way of working, vision, vendor lockout etc.

**Comment by maxvisser** [comment]
I still don't understand the separation of organisation and foundation

**Comment by jamalv** [comment]
Foundation is about the Dutch legal entity and it's employees and activities. 
Organisation is about the "organisation" of the projects.

**Comment by maxvisser** [comment]
Forus / Me Project management & Coordination? :) (https://github.com/ipfs/pm)

**Comment by jamalv** [comment]
Yea what “organisation” is for us, is approximately what the “pm” repository is for ipfs :) 

So it is about the organisation of the projects and processes.

**Comment by jamalv** [comment]
initial version added, will be continually updated
